src/_P095_ILI9341.ino

```diff
@@ -103,23 +103,27 @@ Examples:
 
 //plugin dependency
 #include <Adafruit_ILI9341.h>
+#include "src/Static/Fonts/Seven_Segment24pt7b.h"
+#include "src/Static/Fonts/Seven_Segment18pt7b.h"
+#include "Fonts/FreeSans9pt7b.h"
+
 
 //declare functions for using default value parameters
 void Plugin_095_printText(const char *string, int X, int Y, unsigned int textSize = 1, unsigned short color = ILI9341_WHITE, unsigned short bkcolor = ILI9341_BLACK);
 
-//Define the default values for both ESP32/lolin32 and D1 Mini 
+//Define the default values for both ESP32/lolin32 and D1 Mini
 #ifdef ESP32
 //for D32 Pro with TFT connector
-  #define TFT_CS 14 
+  #define TFT_CS 14
   #define TFT_DC 27
   #define TFT_RST 33
   #define TS_CS 12
 #else
  //for D1 Mini with shield connection
-  #define TFT_CS 16 // D0
-  #define TFT_DC 15 // D8
+  #define TFT_CS 16
+  #define TFT_DC 15
   #define TFT_RST -1
-  #define TS_CS 0   // D3
+  #define TS_CS 0
```

Comment:
```c++
 #define TFT_CS 16 // D0
 #define TFT_DC 15 // D8
 #define TFT_RST -1
 #define TS_CS 0   // D3
```
Mentioning the (strange) NodeMCU names as comment was not such a bad idea.

Reply:
ok